test/apple_shell_testutils.sh

```diff
@@ -449,9 +449,14 @@ function do_action() {
       # Explicitly pass these flags to ensure the external testing infrastructure
       # matches the internal one.
       "--incompatible_merge_genfiles_directory"
-      "--incompatible_objc_compile_info_migration"
   )
 
+  local bazel_version="$(bazel --version)"
```

Comment:
As part of the LTS support, didn't they say there was a way to know which ball version was being used?

As for this, if I run with a non-path bazel, this check will be wrong. Is there a way to get the path to the bazel that is running this?

Reply:
further down, we already invoke bazel directly via `bazel`